docs/localstorage-disks.md

```diff
@@ -0,0 +1,53 @@
+# Local Storage Placement for VM Disks
+
+
+## Introduction
+
+Virtual Machines are able to have a DataVolume disks that are based on Local Storage PVs. Local Storage PVs are bound to a specific node.
+It might happen to have an `Unschedulable` `VMI`, when the PVC is bound to a PV on a different node than virt-launcher pod. 
+
+When the VM with a DataVolumeTemplate is defined a DataVolume is created from the template and the `CDI` creates a worker Pod to import/upload/clone data to the PVC (specified in a template).
+To run a VMI kubevirt creates a virtlauncher pod with all the VMI requirements. Kubernetes uses the virtlauncher pod requirements to schedule it on a specific node.
+Worker Pod might have different constraints than a kubevirt VM. When the VM is scheduled on a different node than the PVC it becomes unusable. 
+This is especially problematic when using a VM with DataVolumeTemplate with many disks managed by CDI. 
+
+This document describes a special handling of `DataVolumes` in the `WaitForFirstConsumer` state (available from [CDI v1.21.0](https://github.com/kubevirt/containerized-data-importer/releases/tag/v1.21.0)).
+
```

Comment:
should this line go at the top of the document since it's the most relevant part of what's being discussed here?

Reply:
Indeed